Code/ForceField/AngleConstraint.cpp

```diff
@@ -69,8 +68,8 @@ AngleConstraintContrib::AngleConstraintContrib(
   d_at3Idx = idx3;
   minAngleDeg += angle;
   maxAngleDeg += angle;
-  RDKit::ForceFieldsHelper::normalizeAngleDeg(minAngleDeg);
-  RDKit::ForceFieldsHelper::normalizeAngleDeg(maxAngleDeg);
+  RANGE_CHECK(0.0, minAngleDeg, 180.0);
+  RANGE_CHECK(0.0, maxAngleDeg, 180.0);
```

Comment:
I think these lines can move inside the `if(relative)` loop

Reply:
line 69-70 definitely, the `RANGE_CHECK` however should stay outside, otherwise if `relative` is `false` we could still set negative values.

Reply:
you can modify minAngleDeg and maxAngleDeg inside the loop too